### PR TITLE
ODS-5543 Claimset metadata IsEdfiPreset and ForApplicationUseOnly is  included in the C# 

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Comment by semalaiappan** [comment]
image
Claimset metadata has IsEdfiPreset and ForApplicationUseOnly now

File: Application/EdFi.Ods.Features/EdFi.Ods.Features.csproj [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.Redis" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.SqlServer" Version="6.0.6" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
Comment (ericstockdale) [inline Application/EdFi.Ods.Features/EdFi.Ods.Features.csproj:23]: @semalaiappan Security.DataAcess package does not appear to be used in this project

Comment (semalaiappan) [reply Application/EdFi.Ods.Features/EdFi.Ods.Features.csproj]: @ericstockdale  removed

**Review by ericstockdale — APPROVED** [review]

File: Application/EdFi.Ods.Repositories.NHibernate.Tests/EdFi.Ods.Repositories.NHibernate.Tests.csproj [hunk 2]
```diff
@@ -19,7 +19,7 @@
   <ItemGroup>
     <PackageReference Include="EdFi.Suite3.Common" Version="6.0.5" />
     <PackageReference Include="Autofac" Version="6.3.0" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Application/EdFi.Ods.Repositories.NHibernate.Tests/EdFi.Ods.Repositories.NHibernate.Tests.csproj]: @ericstockdale  removed it . Thanks

File: Application/EdFi.Ods.Tests/EdFi.Ods.Tests.csproj [hunk 3]
```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Application/EdFi.Ods.Tests/EdFi.Ods.Tests.csproj]: @ericstockdale  removed it . Thanks

File: Application/EdFi.Security.DataAccess/Models/ClaimSet.cs [hunk 4]
```diff
@@ -18,6 +18,10 @@ public class ClaimSet
         [Required]
         public string ClaimSetName { get; set; }
 
+        public bool IsEdfiPreset  { get; set; }
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Application/EdFi.Security.DataAccess/Models/ClaimSet.cs:21]: Thanks

File: Application/Test.Common/Test.Common.csproj [hunk 5]
```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="11.0.1" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Application/Test.Common/Test.Common.csproj]: @ericstockdale  removed it . Thanks

File: tests/EdFi.Ods.WebApi.CompositeSpecFlowTests/EdFi.Ods.WebApi.CompositeSpecFlowTests.csproj [hunk 6]
```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply tests/EdFi.Ods.WebApi.CompositeSpecFlowTests/EdFi.Ods.WebApi.CompositeSpecFlowTests.csproj]: @ericstockdale  removed it . Thanks

File: tests/EdFi.Ods.WebApi.IntegrationTests/EdFi.Ods.WebApi.IntegrationTests.csproj [hunk 7]
```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply tests/EdFi.Ods.WebApi.IntegrationTests/EdFi.Ods.WebApi.IntegrationTests.csproj]: @ericstockdale  removed it . Thanks

**Pull request merged**